ignite/engine/events.py

```diff
@@ -265,7 +265,8 @@ class Events(EventEnum):
 
     - EPOCH_COMPLETED : triggered when the epoch is ended. Note that this is triggered even
       when :meth:`~ignite.engine.engine.Engine.terminate_epoch()` is called.
-    - COMPLETED : triggered when engine's run is completed
+    - COMPLETED : triggered when engine's run is completed or terminated with :meth:`~ignite.engine.engine.Engine.terminate()`,
+      unless the flag `skip_completed` is set to True.
```

Comment:
This line is too long: https://github.com/pytorch/ignite/actions/runs/12130079499/job/33819707813?pr=3309#step:11:80

```suggestion
    - COMPLETED : triggered when engine's run is completed or terminated with
      :meth:`~ignite.engine.engine.Engine.terminate()`, unless the flag
      `skip_completed` is set to True.
```